bch/config/testnet-bch.conf

```diff
@@ -0,0 +1,17 @@
+txindex=1
+dnsseed=0
```

Comment:
Same here
Please try testing the PR locally and checking whether the blockchain height gets updated